main/FatesRestartInterfaceMod.F90

```diff
@@ -107,7 +107,6 @@ module FatesRestartInterfaceMod
   integer, private :: ir_bmort_co
   integer, private :: ir_hmort_co
   integer, private :: ir_cmort_co
-  integer, private :: ir_fmort_co
```

Comment:
I'm wondering if we need to add the new fire mortality arrays to the restarts... I suppose we will see if they were necessary after we run the tests.

Reply:
@rgknox Can you provide more details on this comment? I ran tests across South America with a tropical tree and grass with and with out fire. Both runs completed through 36 years successfully.